src/modules/aws-ec2/shutdown-behavior/setShutdownBehaviorToTerminate.ts

```diff
@@ -0,0 +1,5 @@
+import { changeShutdownBehavior } from "./changeShutdownBehavior";
+
+export const setShutdownBehaviorToTerminate = (instanceId: string) => {
+  changeShutdownBehavior(instanceId, "terminate");
```

Comment:
If this function call is async, might want to just return it here, this will otherwise be a 'fire and forget' call.